WelcomeHome/WelcomeHome.Services/DTO/UserDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WelcomeHome.Services.DTO
+{
+    public class UserDTO
+    {
+        [Required]
+        public string FullName { get; set; } = null!;
+        [Required]
+        public string PhoneNumber { get; set; } = null!;
+        [Required, EmailAddress]
+        public string Email { get; set; } = null!;
```

Comment:
For this we will write validatiors (FluentValidator). Because these conditions are not checked anywhere